src/zip.js

```diff
@@ -3,7 +3,14 @@ var write = require("./write"),
   prj = require("./prj"),
   JSZip = require("jszip");
 
-module.exports = function (gj, options, stream = false) {
+module.exports = function (
+  gj,
+  options = {
```

Comment:
We also need to add the types field, which could probably be init to...
```typescript
types: {
  point: "mypoints",
  polygon: "mypolygons",
  line: "mylines",
},
```        

when I leave the options undefined to try to test the defaults, I get this error:

``` console
TypeError: Cannot read properties of undefined (reading 'point')
    at zip.js:34:32
    at write (write.js:49:5)
    at zip.js:25:7
    at Array.forEach (<anonymous>)
    at module.exports (zip.js:23:5)
```